### ISSUE TITLE
Invoice Not getting paid in new version

### ISSUE THREAD

**Issue by cystec** [razorpay/razorpay-whmcs#26]
Hello,

Just updated the razorpay module...now some invoices are not getting paid.

getting error

Unsuccessful-WHMCS_ERROR: Payment to Razorpay Failed. Either razorpay_order_id or razorpay_subscription_id must be present.. Please check razorpay dashboard for Payment id: pay_E**********
--

**Comment by worldofsmarty** [comment]
Same here... We are getting the same error in the payment gateway logs...

**Comment by systron** [comment]
Yes, Our Invoices are also not getting paid after latest update version of razorpay by KDC
Getting these error:
Unsuccessful-BAD_REQUEST_ERROR:Capture amount must be equal to the amount authorized. Please check razorpay dashboard for Payment id: pay_FDZVe5K1jPAgeu

its effecting serious transaction issues. secondly Merchant also not getting the email of payment recived on WHMCS order.

**Comment by hemangjoshi37a** [comment]
Any update? is it solved or not yet??
please help...

**Comment by systron** [comment]
Hemangbhai, It's resolved.




On Mon, Aug 24, 2020 at 5:25 PM +0530, "Hemang Joshi" <notifications@github.com> wrote:















Any update? is it solved or not yet??

please help...
